README.md

```diff
@@ -1,8 +1,8 @@
 # UP
 
-UP is a simple client for testing Prometheus remote-write requests.
-The client writes the specified metric at a chosen interval, where the value of the metric is always the current timestamp in milliseconds.
-It can also read the metric back from a specified endpoint and compare its value against the current time to determine the total write-read latency.
+UP is a simple client for testing Prometheus remote-write and Loki write requests.
+- For a specified metric the client writes the metric at a chosen interval, where the value of the metric is always the current timestamp in milliseconds. It can also read the metric back from a specified endpoint and compare its value against the current time to determine the total write-read latency.
```

Comment:
nit: can we keep each sentence on a new line? It helps keep the markdown easy to review when one since word changes in a sentence, rather than mark the whole paragraph as a diff